elide-integration-tests/src/test/java/com/yahoo/elide/tests/ResourceIT.java

```diff
@@ -737,8 +737,8 @@ public void testForbiddenDeleteEmptyCollectionRelationship() throws Exception {
 
     @Test(priority = 11)
     public void testDeleteParent() {
+        //Should work without content-type and body
         given()
```

Comment:
@DeathByTape I'm using resteasy - not jersey in my project. I've checked this test again and I've found that it's my fault. This test actually passes with and without content-type header on both code versions - original and this one from pull request. Obviously we need new representative test to cover this case.

This bug is probably specific to resteasy JAX-RS implementation. I've created modified blog example using resteasy instead of jersey and `curl -X DELETE http://localhost:4080/user/2` fails with `415 Unsupported Media Type` with original elide-core code and deletes successfuly (`204 No Content`) with my changes form this pull request.